katas/sort_cards.py

```diff
@@ -0,0 +1,37 @@
+"""The sort cards kata from CodeWars."""
+
+
+def sort_cards(cards):
+    """Sort shuffled list of cards, sorted by rank.
+
+    >>> sort_cards(['3', '9', 'A', '5', 'T', '8', '2', '4', 'Q', '7', 'J', '6', 'K'])
+    ['A', '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K']
+
+    """
+    cards = sorted(list(cards))
```

Comment:
you went thru and sorted, but then you do these .count loops for all of the other cards.  This is incredibly inefficient.  Look into the key parameter for sorted.